### PR TITLE
Add ability to disable triggers

### PR DESCRIPTION
**Pull request by samollari**
IG trigger isn't working right and multiple instances of the bot running simultaneously certainly aren't helping. This allows disabling triggers so they don't run twice

### CODE REVIEW

File: config.json.example [hunk 1]
```diff
@@ -4,5 +4,8 @@
     "memberCountGuild": {
         "guild": "guild_id",
         "channel": "channel_id"
-    }
+    },
+    "runtimeFlags": [
+        "DISABLE_TRIGGERS"
+    ]
```
Comment (Muirrum) [inline config.json.example]: Is there a way to make this a cli flag? I think i'd prefer making this something that can be disabled without needing to edit a file. Maybe using environment variables?

**Review by Muirrum — CHANGES_REQUESTED** [review]

File: config.json.example (hunk 1 above)
Comment (samollari) [reply config.json.example]: Yeah I can. The only reason I didn't is so you don't have to pass the args every time

Comment (Muirrum) [reply config.json.example]: Yeah I see that, I'm just envisioning us being forgetful and not changing that when we need to.

Comment (samollari) [reply config.json.example]: Sometime soon I'm going to revisit this and implement running with CLI flags because I do believe that is the right approach

**Pull request closed without merging**